exporters/common/src/main/java/io/opentelemetry/exporter/internal/http/HttpExporter.java

```diff
@@ -77,7 +77,14 @@ public CompletableResultCode export(T exportRequest, int numItems) {
           try {
             body = httpResponse.responseBody();
           } catch (IOException ex) {
-            throw new IllegalStateException(ex);
+            logger.log(
+                Level.WARNING,
+                "Failed to export "
+                    + type
+                    + "s. Server responded with exception message "
+                    + ex.getMessage());
+            result.fail();
+            return;
```

Comment:
```suggestion
            logger.log(Level.FIND, "Unable to obtain response body", ex);
```
We should just indicate that something went wrong extracting the request body but fall through to the code which follows to log the status code and call `result.fail()`.

Reply:
just changed this to FINE